src/assets/server/rest/v1/schemas/chargingstation/chargingstation-action.json

```diff
@@ -0,0 +1,96 @@
+{
+  "title": "Charging Station Action Request",
+  "type": "object",
+  "properties": {
+    "chargeBoxID": {
+      "$ref": "chargingstation.json#/definitions/id"
+    },
+    "carID": {
+      "$ref": "common.json#/definitions/id"
+    },
+    "args": {
```

Comment:
For the charging station's action it's more tricky.
We have one endpoint and we mixed everything unfortunately!
I would like that we match the OCPP standard according the Action and that we identify the mandatory params in the 'args' object.
We can have a meeting for this and let it as it is for the time being.

Reply:
Sure, as you have a much busier timetable than me, can I let you plan it ?